src/mokka/src/main/resources/application.properties

```diff
@@ -3,6 +3,6 @@ redirectURL=
 notificationURL=
 paymentSchema=http://TODO/service/PaymentNotificationService
 file.download.directory=
-
+contentDisposition=inline
```

Comment:
I would say "attachment" is much better default option.

Should not we prepend property name with "files." to make it more descriptive?